src/OfficeToolPlus/Dictionaries/Languages/ru-ru.xaml

```diff
@@ -98,153 +98,153 @@
+    <sys:String x:Key="Deploy_Advanced_Migrate_Arch.Description">Изменить существующую установку Office на текущую указанную архитектуру. Например, с 32-битной на 64-битную.</sys:String>
     <sys:String x:Key="Deploy_Advanced_Remove_MSI">Удаление существующих MSI версий Office.</sys:String>
     <sys:String x:Key="Deploy_Advanced_Match_Previous_MSI">Использовать язык от предыдущей MSI версии.</sys:String>
-    <sys:String x:Key="Deploy_Advanced_Ignore_Product">Ignore this products</sys:String>
+    <sys:String x:Key="Deploy_Advanced_Ignore_Product">Игнорировать эти продукты</sys:String>
     
     <sys:String x:Key="Deploy_Advanced_Activate_Settings">Настройки активации</sys:String>
-    <sys:String x:Key="Deploy_Advanced_Accept_EULA">Automatically accept the EULA</sys:String>
-    <sys:String x:Key="Deploy_Advanced_Accept_EULA.Description">When enabled, the "Microsoft Software License Terms" dialog box will not be displayed.</sys:String>
+    <sys:String x:Key="Deploy_Advanced_Accept_EULA">Автоматически принять лицензионное соглашение</sys:String>
+    <sys:String x:Key="Deploy_Advanced_Accept_EULA.Description">Если этот параметр включен, диалоговое окно «Условия лицензионного соглашения на программное обеспечение Microsoft» отображаться не будет.</sys:String>
     <sys:String x:Key="Deploy_Advanced_MAK">Настройки MAK</sys:String>
     <sys:String x:Key="Deploy_Advanced_Auto_Activate">Мы попробуем автоматически активировать продукт после настройки MAK.</sys:String>
-    <sys:String x:Key="Deploy_Advanced_Product_Activation">Product activation</sys:String>
-    <sys:String x:Key="Deploy_Advanced_UserBased">User based</sys:String>
-    <sys:String x:Key="Deploy_Advanced_DeviceBased">Device based</sys:String>
-    <sys:String x:Key="Deploy_Advanced_SCL">Shared computer</sys:String>
-    <sys:String x:Key="Deploy_Advanced_SCL_Cache_Override">Overwrite licensing token location</sys:String>
-    <sys:String x:Key="Deploy_Advanced_SCL_Cache_Override.ToolTip">Specify a folder location for the licensing token.</sys:String>
+    <sys:String x:Key="Deploy_Advanced_Product_Activation">Активация продукта</sys:String>
+    <sys:String x:Key="Deploy_Advanced_UserBased">Пользователем</sys:String>
+    <sys:String x:Key="Deploy_Advanced_DeviceBased">Устройством</sys:String>
+    <sys:String x:Key="Deploy_Advanced_SCL">Общим компьютером</sys:String>
+    <sys:String x:Key="Deploy_Advanced_SCL_Cache_Override">Перезаписать расположение токена лицензирования</sys:String>
+    <sys:String x:Key="Deploy_Advanced_SCL_Cache_Override.ToolTip">Укажите расположение папки для токена лицензирования.</sys:String>
 
     <sys:String x:Key="Deploy_Advanced_Preferences">Настройки программ</sys:String>
     <sys:String x:Key="Deploy_Advanced_Apply_Preferences.Description">Apply application preferences to Office applications immediately.</sys:String>
```

Comment:
Still not finished? And line 251, 253, 254, 261, 262.

Reply:
Updated this area. If possible, agreed with the official translation https://config.office.com/deploymentsettings

Reply:
I recommend adding links, as done in the official version.
https://go.microsoft.com/fwlink/p/?linkid=2094112
https://go.microsoft.com/fwlink/p/?linkid=2094114
https://go.microsoft.com/fwlink/p/?linkid=2094216
![2023-01-02 20 43 42](https://user-images.githubusercontent.com/19782385/210253313-d6b2b2b3-2b5c-4dbd-b40a-8aa4e8208edd.png)

Reply:
The line 740. I did not translate, because above there are similar lines with instructions not to translate.